addons/account/views/account_move_views.xml

```diff
@@ -1105,7 +1105,16 @@
                                         <field name="credit"
                                                sum="Total Credit"
                                                attrs="{'invisible': [('display_type', 'in', ('line_section', 'line_note'))]}"/>
-                                        <field name="tax_tag_ids" widget="many2many_tags" string="Tax Grids" optional="show" domain="[('country_id', '=', parent.tax_country_id), ('applicability', '=', 'taxes')]"/>
+
+                                        <field name="tax_tag_ids"
+                                               widget="many2many_tags"
+                                               string="Tax Grids"
+                                               optional="show"
+                                               domain="['|', ('country_id', '=', parent.tax_country_id),
+                                                             ('country_id', '=', False),
+                                                         ('applicability', '=', 'taxes')]"
```

Comment:
you can't indent the domain like this because it makes the `ir.ui.view` really weird :/